travis/exec.yml

```diff
@@ -0,0 +1,4 @@
+before_install:
+  - mkdir -p .ci && curl -sL https://github.com/logstash-plugins/.ci/archive/travis_refactor.tar.gz | tar zxvf - --skip-old-files --strip-components=1 -C .ci --wildcards "*Dockerfile*" "*docker*" "*.sh"
```

Comment:
the decision left is whether we point everything to master or start with plugins importing `travis.yml` from a tag.
master should be fine as a starter (we might still want to tag `v0.1.0` or whatever and sync all places to use tag):
- travis/exec.yml `curl -sL https://github.com/logstash-plugins/.ci/archive/{master|v0.1.0}.tar.gz`
- travis/travis.yml `logstash-plugins/.ci:travis/defaults.yml@{master|v0.1.0}`
- README to rather mention master (or a tag) e.g. 
```yaml
import:
- logstash-plugins/.ci:travis/defaults.yml@master
# ...
```

Reply:
++ my plan is to merge this into master and then push a v0.1.0 commit + tag